LeetcodeProblems/Algorithms/easy/Happy_Number.js

```diff
@@ -40,7 +49,7 @@ function checkHappyNumber(n){
   let strNumber = n.toString();
   let splitNumber = strNumber.split("");
   if(splitNumber.length <= 1){
-    return (n <= 1)? true:false;
+    return (n === 1 || n === 7)? true:false;
```

Comment:
I don't think we'd need to check for `7` since it's already covered by `n===1`. 

```suggestion
    return n === 1;
```

Reply:
Hi,
Thanks for getting back to me, the thing is the first `if` condition checks whether the array's length is 1 or not, 
in this case if the array has only `7` then it would be true, and since it is to return `true` if `n===1` it would return false.

I have added some screenshots for tests please feel free to check them.


- Not adding check for `n===7`
![Screenshot 2024-02-12 at 04-33-01 Happy Number - LeetCode](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/assets/85690382/ac3c1cb8-5824-4c2e-82b2-0c200a29a88e)

- Adding checks
![Screenshot 2024-02-12 at 04-33-43 Happy Number - LeetCode](https://github.com/ignacio-chiazzo/Algorithms-Leetcode-Javascript/assets/85690382/7d7213ff-6c3c-441a-9ab8-4d2023b980d7)

Reply:
ohhhh true! Nice catch. We need to list all happy numbers that are less than 10.